.vimbundle

```diff
@@ -9,59 +9,205 @@ end
 
 call plug#begin('~/.vim/bundle')
 
-" Make sure you use single quotes
-Plug 'AndrewRadev/splitjoin.vim'
-Plug 'adamlowe/vim-slurper'
+" {{{ Language Support Plugins
+Plug 'vim-ruby/vim-ruby'
 Plug 'elixir-lang/vim-elixir'
-Plug 'ervandew/supertab'
 Plug 'exu/pgsql.vim'
-Plug 'godlygeek/tabular'
-Plug 'gregsexton/gitv'
-Plug 'hashrocket/vim-hashrocket'
 Plug 'heartsentwined/vim-emblem'
+Plug 'pangloss/vim-javascript'
+Plug 'slim-template/vim-slim'
+Plug 'tpope/vim-haml'
+Plug 'tpope/vim-markdown'
+Plug 'leshill/vim-json'
+Plug 'MaxMEllon/vim-jsx-pretty'
+" }}}
+
+" {{{ Themes
+Plug 'jgdavey/vim-railscasts'
+Plug 'therubymug/vim-pyte'
+Plug 'wgibbs/vim-irblack'
+Plug 'tpope/vim-vividchalk'
+" }}}
+
+" {{{ GIT Plugins
+" No longer maintained by this person
+Plug 'gregsexton/gitv'
+" Git plugin so awesome it should be illegal
+" Everyone should really learn how to use this better.
+Plug 'tpope/vim-fugitive'
+" Fairly active, but with no README
+Plug 'tpope/vim-git'
+" }}}
+
+" Primary function
+" gS - split a one liner into multiple lines
+" gJ - do the opposite
+Plug 'AndrewRadev/splitjoin.vim'
+
+" Used with pivotal stories for a bulk import
+ Plug 'adamlowe/vim-slurper'
+
+" Support tab completion
+" TODO: is there a better tab completion plugin?
+Plug 'ervandew/supertab'
+
+" Allows you to easily line things up like so:
+" a = 1
+" back = 3
+"
+" Becomes
+" a    = 1
+" back = 3
+"
+" Which is a pattern that disgusts me. So I will be removing this to an
```

Comment:
HAHA 😂

Reply:
Oh shoot, Lol. I forgot to moderate my snark.  I had originally planned for this to just be my modified version of dotmatrix. 😨